backend/app/minio/utils.py

```
@@ -0,0 +1,96 @@
+from fastapi import UploadFile, HTTPException
+
+from fastapi import UploadFile
```

Ruff (0.8.2)

Redefinition of unused `UploadFile` from line 1

Remove definition: `UploadFile`

(F811)